plugins/kubernetes/test/models/watchers/deploy_watcher_test.rb

```diff
@@ -9,6 +9,9 @@
   let(:project) { current_release.project }
 
   before do
+    # Disable multithreading so celluloid stays in the same transaction
+    ActiveRecord::Base.stubs(connection: ActiveRecord::Base.connection)
```

Comment:
hmm that blogger sounds familiar, should we report the soft_deletion issue upstream to him as well?